plant/plants/api/serializers.py

```diff
@@ -1,11 +1,31 @@
 from rest_framework import serializers
 
-from plant.plants.models import Plant
+from plant.plants.models import Plant, Watering
+
+
+class PlantWateringSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Watering
+        fields = ["plant", "litres", "watering_date"]
 
 
 class PlantSerializer(serializers.ModelSerializer):
     is_watered = serializers.BooleanField()
+    watering = PlantWateringSerializer(many=True)
```

Comment:
Please change form to the plural, add read_only=True keyword parameter to the serializer, as we will not support adding new watering through this serializer